releasenotes/notes/deterministic-barrier-before-final-measurements-04e817d995794067.yaml

```diff
@@ -0,0 +1,9 @@
+---
+fixes:
+  - |
+    The :class:`.BarrierBeforeFinalMeasurements` transpiler pass previously had a non-deterministic
```

Comment:
Do you want to also mention `MergeAdjacentBarriers` here too, since it can also be an issue with the merge pass if it's run on its own without `BarrierBeforeFinalMeasurements`.

Reply:
Done in fbfada661aa3c273f2f80566a48e301d2767a9b2.